workspaces/redhat-resource-optimization/docs/adrs/0001-optimizations-plugin-for-red-hat-developer-hub-phase-1.md

```diff
@@ -0,0 +1,52 @@
pfVJ2CiuOVaeb9Vzl7fbNp8u2xZSdJlSSCkue0qnF5c/fXl98nrtC8m09DoIKUAZISVAnO4BsmxD3aFHT+pQ+n90Nk2CQrtyHzjEjreyy6WW1eWl46tfv5xikuGGU8m09HL5ifM0pGRuJmdf707ts1wiTUgBFCElQIQUIMs21DEjpCAWhJQA0ScFyLINdcwIKYgFISVA3BYfyLINdcwIKYgFISVAnO4BsmxDHTNCCmJBSAkQIQXIsg11zAgpiAUhBUBDsA11zAgpiAUhJUD0SQGyfAOdu36EK647kWm4q7Lz7UxZfvzMTFlHDwGUm67ZMrlZnC3T8tS9U6qlDy7MF/Q+L/kxU9tNI6QgFoSUAHG6B8jyDbSEFNug5ybOK49vOeMK09fozdy0vNinPJQ7wm75kY7rvUxKISU/bVX5pm9ywza56VpbSNEHBrbd/8SHFHlwoF9nS/8hGhokmOSHjS+XtYUUP4/cUTZ9Izjdh7HT9Jk/uQE36kMOZZ8Ksx/VoawvV9p+svz8rTqfv7kcIQWxIqQEiJACZKUb6XbaboImwcCXFTf+oDzub7YmT0HedDq7bPrmbW3T9aZqbU8uToYidXdZ3cZD3y3fxC31hOXMjeK8tpu9daYwd5MGk+IVntkj0/04IQWxIKQAaAi20Y4ZIQWxIKQEiCMpQJZtqCtVvH9fpkye62PLkvk3/qDD6Xrq5gpHOjojyxUW771clrrFfncRUhALQkqACClAlm2oJTDoQwDl6cHydOK2YJG//V6XlycOl17nR01y+Zvvci3FPno6qPjAfp3H9xeReeRpxPJ05Pyts3WajOs0eVDhhDnJ9vRBgQ8+4wpLD2qfFllWH0y4YFt5ng2nXH7stHLfktJyLQOHJqeLpG+LhpXpa8r9TYaMvtxn5bE3XK40r/SL0ddtp51yN9xc3oZ5wjIhBTEhpASI2+IDWbah1oZcwkep0S9IgLhvZxJANKQMHqlX4viOsX6aH7b06acBJTfi9vbrlfBzzYBycChNT0+TTrOyjI5LQJKnK8sDAWW9pZCiTzkuhRtZLh2ItCOtLLfrHX2tIUXmkyBU2r4PLBK6cqX98svJe/L7nEZIQSwIKQAagm2oRc2XIl9FxSUHMmXdRUhBLAgpAeJ0D5BlG+qYEVIQC0JKgAgpQJZtqOvJ34+kM/nRk3VYmLK0fEqm7fRRbyGkIBaElADRJwXIsg11PUkfFB1vO30kfVikA2zupol6ZY70X5EOrdoHpTT0IUWma9+R+/fp/Ol7pvQkQgpiQUgJELfFB7JsQ11v/kZsxTXHysMK+rsUlx5MOtL6G7HJbeztfPVGSEEsCCkB4nQPkGUb6pgRUhALQkqACClAlm2oY0ZIQSwIKQAagm2oY0ZIQSwIKQGiTwqQZRvqmI3ffSZTP0AzIqQEiNM9QMcGP/P7TIMdm8mv/WumXoBmRUgJECEFAABCCgAACBQhJUAcSQEAgJASJEIKAACElCBxW3wAAAgpAAAgUISUAHG6BwAAQkqQCCkAABBSgkSfFAAACClB4rb4AAAQUoLE6R4AAAgpQSKkAABASAFq9h//8R/Klteqs/UuWbKkw/Irueeee7pcxm9v5cqVV5z37NmzmTIA6AmElADRJ6WxvPvuuzr88MMP3cyZM92pU6f09Z/+9Ccdzp0713388cc6vn37drdixQo3cuRId++992rZRx99pMPPPvvMbdmyJVnvvHnzdCjLyjb+8R//0S1evFgDhN/moUOH3MaNG90nn3zSbn/Eiy++6N5++20dyrJ+GVl+/Pjx7qc//am7dOlSspwPJj6sLFy4MFnfrFmz9L3Je3niiSd0WVmnXxYAegIhJUCc7mks0tBLELj77ruTBl4Chgxvu+02HY4aNcodPnxY59uxY4cbO3ase+yxx9odsfDL2vWny4cNG5aZT8a//fbbdq/tcqdPn9bxTz/9tN20iRMnujVr1iRlX3zxhYYRGV+3bl27eSdNmqRHWeRIil+vvB+7vwBQL4SUABFSGos/QvH0008nDfr777/vPv/8c7dv3752YcGPS0iRcPDKK6/oax9m/HzpeW+44YYO15Gez4aUtHTZmDFjdF7/2oaU9DokWMnRoPS6CCkAriZCCtAA0gGiN8j2v/76azdixIjMNADoKYSUAHEkBQAAQkqQCCmNp6MjHdLnxJZ1tQwA4DJCSoC4LX5jOXjwYLvXa9eudT/60Y+078emTZvc0aNH3UsvvaTTpJ/Ke++9p+PpfiQAgCxCClAH/qiIdDSVkCJX07z++ut6ma6U+ZAiQ3+5MCEFAK6MkBIgTvfE4a677sqUAQAuI6QEiJACAAAhJUj0SQmf9DWxZQCA+iKkBIjb4gNA1/itbH6ElABxugcAAEJKkAgpAAAQUgAAQKAIKQHiPCsAAISUIHG6BwAAQkqQCCkAABBSAABAoAgpAeJICgAAhJQgEVIAACCkBMneFl+esOufstuRI0eOZMq+/vrrTBkAAI2EkNIApk2b5k6ePKnjPrD86U9/SoKLhBQZ3717tw5HjBihwxUrViTzpIOODD/88MPMdgAACAkhJUAdne5ZtmyZDn3YSB9Z8eO7du3S8aFDh+qRlHvuuUdfjxw50u3bt0/Hd+7cmVkeAIAQEVICZENKOlRI+PCv5WiKlPkjKfL6o48+cnPnztXXn3/+eTJPeh3nz58npAAAgkdICZDtk1IPHD0BADQaQkqAuC0+AACElCClT/esX79eT/FIPxM73/Tp0zNlabfddlumDACARkFICZANKf5KnGeeeUaHn3zyiV7JIyHl8OHDybRvv/3W7dixw50+fVrLCCkAgEZGSAmchJTZs2drB1h/afHPf/5zDSlSJkdZpkyZ4vbv36/jElRGjRqlyxJSAACNjJASoHSfFB84qnXjjTd2eIoIAIBGQUgJ0KZNmzJlAADEhpACAGgK/IHXfAgpAeM/HABUhls3NKf/D7yCTHT7eJaOAAAAAElFTkSuQmCC
```

Comment:
This image doesn't seem to show up, I would expect to see it here at the bottom:

<img width="1164" alt="Screenshot 2024-12-06 at 10 08 14 AM" src="https://github.com/user-attachments/assets/9a4b50a4-95b9-4fff-9b3c-62855b246079">

Reply:
> One question about the image but I also have some more general concerns about this that I would like to discuss at the next Community Plugins SIG.

Fixed in 9e993324d87baba60cf857f1fc18deaea3d13a7a